app/assets/javascripts/components/admin/audits.jsx

```diff
@@ -0,0 +1,21 @@
+// Audits component; makes up the root audits view.
+class Audits extends React.Component {
+    constructor(props) {
+      super(props);
+      this.state = this.props;
+    }
+  
+    render() {
+      return (
+        <div className="pt-3">
+          <Breadcrumb
+            crumbs={[{ name: 'Audits', url: Routes.reports_path() }]}
```

Comment:
I think this might be missing a few levels, it used to look like:
```
crumbs: [{ name: 'Dashboard', url: death_records_path }, { name: 'Administration', url: admins_path }, { name: 'Audit Report' }]
```